tagsets/expregister.php

```diff
@@ -335,11 +335,23 @@ function expregister__cancel($participant,$session) {
     $done=experimentmail__experiment_cancellation_mail($participant,$session);
 }
 
-
-
+function expregister__language_test_signed_select_field($postvarname,$selected,$hidden=array(),$show_color=true) {
+    $language_test=[ 'y', 'n', 'o' ];
+    $out='<SELECT name="'.$postvarname.'">';
+    foreach ($language_test as $test_value) {
+        if (!in_array($status['language_test'],$hidden)) {
```

Comment:
Where does $status come from?

Reply:
copied from rules signed function. might me useless. however, it seems to work.